039-AKSForEnterprise/Coach/08-arc.md

```diff
@@ -0,0 +1,280 @@
+# Challenge 8. Arc enabled Kubernetes and Arc enabled data Services - Coach's Guide

```

Comment:
Need to fix some capitalization here:

"Challenge 8: Arc Enabled Kubernetes and Arc Enabled Data Services - Coach's Guide"

Reply:
Done! (and added hyphen)